crowbar_framework/app/models/service_object.rb

```diff
@@ -822,9 +822,9 @@ def apply_role(role, inst, in_queue)
     # We also check that all nodes we'll require are in the ready state.
     #
 
-    # Initialize variables used in ensure at the end of the method
-    chef_daemon_nodes = []
-
+    # get all nodes except the admin node
+    nodes_without_admin = NodeObject.find_all_nodes.reject { |n| n.admin? }.map(&:name)
```

Comment:
Style/SymbolProc: Pass `&:admin?` as an argument to `reject` instead of a block.